### ISSUE TITLE
volume_grpc_client_to_master.go:135] Volume Server found a new master newLeader: 0.0.0.0:9333 instead of master:9333

### ISSUE THREAD

**Issue by SvenDowideit** [chrislusf/seaweedfs#1807]
My single volume server keeps finding itself as the master, and not the one specified in the commandline.

the master and volume servers are in separate containers, using docker networking, but the result of bouncing the docker daemon is suggesting that the volume server is deciding that it is also the master.

this is not helpful, as the remainder of the system then gets "out of volumes" errors.

```
services:
  master:
    hostname: "{{.Node.Hostname}}_{{.Service.Name}}"
    image: chrislusf/seaweedfs:2.24
    command: "master -ip 0.0.0.0 -port 9333 -defaultReplication=000 -metrics.address=pushgateway:9091"
    networks:
      - internal
    # volumes:
    #   - data:/data
    deploy:
      replicas: 1

  volume:
    hostname: "{{.Node.Hostname}}_{{.Service.Name}}"
    image: chrislusf/seaweedfs:2.24
    command: 'volume -max=50 -mserver=master:9333 -port=8080'
    networks:
      - internal
    volumes:
      - data:/data
    deploy:
      mode: global

```

**Comment by SvenDowideit** [comment]
oho - so removing the `-ip 0.0.0.0` from the master changes this (i'm thinking fixes it) - I don't recal what this was added to fix way back in the seaweedfs 1.4 deployment - but I really do wish seaweedfs could be forced into not storing or passing around ip addresses (yeah, I realise there are good reasons)

**Comment by chrislusf** [comment]
correct -ip setting is required.
